clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoOptions.java

```diff
@@ -81,6 +87,11 @@ public Options options() {
             "z", PROVIDER, "provider one of hadoop, hive, mysql, postgres, iceberg, kafka"));
     options.addOption(createArgOption("l", USER, "user name"));
     options.addOption(createArgOption("g", GROUP, "group name"));
+    options.addOption(createArgOption(DATATYPE, "column data type"));
+    options.addOption(createArgOption(POSITION, "position of column"));
+    options.addOption(createArgOption(NULL, "column value can be null (true/false)"));
```

Comment:
I suggest to use "nullable", not "null", because "null" is a noun, means the value is no represented. "nullable" means this filed can be empty, can be non-empty. They are different.

Reply:
Null is an adjective, not a noun. A column can be nullable, but can have a null value or allow nulls. I selected NULL as that is similar to SQL statements e.g. "...ALTER COLUMN columnName datatype NULL"

Reply:
I see, it is okay to use "null".